domains/Discount.js

```diff
@@ -0,0 +1,60 @@
+import MENU_BOARD from '../constants/menu.js';
+import { RANGE, DATE, PRICE } from '../constants/promotion.js';
+import { checkIsInRange, checkIsNumber } from '../utils/ValidationUtils.js';
+
+class Discount {
+  #date;
+
+  constructor(date) {
+    this.#date = date;
+  }
+
+  static validate(date) {
+    if (!checkIsNumber(date) || !checkIsInRange(RANGE.min, RANGE.max, date))
+      return false;
+    return true;
```

Comment:
```suggestion
      return checkIsNumber(date) && checkIsInRange(RANGE.min, RANGE.max, date))
```
이런식으로도 표현할 수 있을 것 같아요!

Reply:
아차차 !! 드모르간 법칙 인가요 !!